examples/charts/file/timeseries.py

```diff
@@ -0,0 +1,51 @@
+from collections import OrderedDict
+
+import pandas as pd
+
+from bokeh.charts import TimeSeries, show, output_file, defaults, vplot, hplot
+
+# read in some stock data from the Yahoo Finance API
+AAPL = pd.read_csv(
+    "http://ichart.yahoo.com/table.csv?s=AAPL&a=0&b=1&c=2000&d=0&e=1&f=2010",
+    parse_dates=['Date'])
+MSFT = pd.read_csv(
+    "http://ichart.yahoo.com/table.csv?s=MSFT&a=0&b=1&c=2000&d=0&e=1&f=2010",
+    parse_dates=['Date'])
+IBM = pd.read_csv(
+    "http://ichart.yahoo.com/table.csv?s=IBM&a=0&b=1&c=2000&d=0&e=1&f=2010",
+    parse_dates=['Date'])
+
+xyvalues = OrderedDict(
+    AAPL=AAPL['Adj Close'],
+    Date=AAPL['Date'],
+    MSFT=MSFT['Adj Close'],
+    IBM=IBM['Adj Close'],
+)
```

Comment:
Does this  need to be an `OrderedDict`? It's probably an unusual data structure for many users. Will a simple `dict` work? (I am not sure anything about the order here matters?) 

Also, if the time series happen to line up, would this work with a `DataFrame` like all the other new examples?